config/notifiers.go

```diff
@@ -394,3 +401,45 @@ func (c *PushoverConfig) UnmarshalYAML(unmarshal func(interface{}) error) error
 	}
 	return checkOverflow(c.XXX, "pushover config")
 }
+
+// StatusPageConfig configures notifications via StatusPage.
+type StatusPageConfig struct {
+	NotifierConfig `yaml:",inline"`
+
+	APIURL      string `yaml:"api_url"`
+	APIKey      Secret `yaml:"api_key"`
+	PageId      string `yaml:"page_id"`
+	ComponentId string `yaml:"component_id"`
+	Status      string `yaml:"status"`
```

Comment:
When [updating a component](https://doers.statuspage.io/api/v1/components/) you can as well pass a new `name` or `description`, but those two values are not related to the component's status. So the given five parameters are the ones needed for updating a components status to `operational`, `degraded_performance`, `partial_outage` or `major_outage`.

Which additional parameters do you see in the API docs that users may want to control?